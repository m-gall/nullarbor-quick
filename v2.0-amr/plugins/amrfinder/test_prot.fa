>blaTEM-156
MSIQHFRVALIPFFAAFCLPVFAHPETLVKVKDAEDQLGARVGYIELDLNSGKILESFRPEERFPMMSTFKVLLCGAVLS
RVDAGQEQLGRRIHYSQNDLVEYSPVTEKHLTDGMTVRELCSAAITMSDNTAANLLLTTIGGPKELTAFLHNIGDHVTRL
DRWEPELNEAIPNDERDTTMPAAMATTLRKLLTGELLTLASRQQLIDWMEADKVAGPLLRSALPAGWFIADKSGAGERGS
RGIIAALGPDGKPSRIVVIYTTGSQATMDERNRQIAEIGASLIKHW
>blaPDC-114_blast BLAST (100% length, but 1 mismatch)
MRDTRFPCLCGIAASTLLFATTPAIAGEAPADRLKALVDAAVQPVMKANDIPGLAVAISLKGEPHYFSYGLASKEDGRRV
TPETLFEIGSVSKTFTATLAGYALTQDKMRLDDRASQHWPALVGSRFDGISLLDLATYTAGGLPLQFPDSVQKDQAQIRD
YYRQWQPTYAPGSQRLYSNPSIGLFGYLAARSLGQPFERLMEQQVFPALGLEQTHLDVPEAALAQYAQGYGKDDRPLRAG
PGPLDAEGYGVKTSAADLLRFVDANLHPERLDRPWAQALDATHRGYYKVGDMTQGLGWEAYDWPISLKRLQAGNSTPMAL
QPHRIARLPAPQALEGQRLLNKTGSTNGFGAYVAFVPGRDLGLVILANRNYPNAERVKIAYAILSGLEQQGKVPLKR
>blaOXA-436_partial (Should be partial OXA-48 family
MRALALSAVLMVTTMIGMPAVAKEWQENKSWNAHFSEHKTQGVVVLWNENTQQGFTNDLKRANQAFLPASTFKIPNSLIA
LDLGVVKDEHQVFKWDGQTRDIAAWNRDHDLITAMKYSVVPVYQEFARQIGEARMSKMLHAFDYGNEDISGNLDSFWLDG
GIRISATQQIAFLRKLYHNKLHVSERSQRIVKQAMLTEANADYIIRAKTGYSVRIEPKIGWWVGWIELDDNVW
>vanG
MQNKKIAVIFGGNSTEYEVSLQSASAVFENINTNKFDIIPIGITRSGEWYHYTGEKEKILNNTWFEDSKN
LCPVVVSQNRSVKGFLEIASDKYRIIKVDLVFPVLHGKNGENGTLQGIFELAGIPVVGCDTLSSALCMDK
DRAHKLVSLAGISVPKSVTFKRFNEEAAMKEIEANLTYPLFIKPVRAGSSFGITKVIEKQELDAAIELAF
EHDTEVIVEETINGFEVGCAVLGIDELIVGRVDEIELSSGFFDYTEKYTLKSSKIYMPARIDAEAEKRIQ
EAAVTIYKALGCSGFSRVDMFYTPSGEIVFNEVNTIPGFTSHSRYPNMMKGIGLSFSQMLDKLIGLYVE
>gyrA  T86I  Campylobacter
MENIFSKDSDIELVDIENSIKGSYLDYSMSVIIGRALPDARDGLKPVHRRILYAMQNDEAKSRTDFVKSA
RIVGAVIGRYHPHGDIAVYDALVRMAQDFSMRYPSITGQGNFGSIDGDSAAAMRYTEAKMSKLSHELLKD
IDKDTVDFVPNYDGSESEPDVLPSRVPNLLLNGSSGIAVGMATNIPPHSLNELIDGLLYLLDSKDASLEE
IMQFIKGPDFPTGGIIYGKKGIIEAYRTGRGRVKVRAKTHIEKKTNKDVIVIDELPYQTNKARLIEQIAE
LVKEKQIEGISEVRDESNKEGIRVVIELKREAMSEIVLNNLFKSTTMESTFGVIMLAIHNKEPKIFSLLE
LLNLFLTHRKTVIIRRTIFELQKARARAHILEGLKIALDNIDEVIALIKNSSDNNTARDSLVAKFGLSEL
QANAILDMKLGRLTGLEREKIENELAELMKEIARLEEILKSETLLENLIRDELKEIRSKFDVPRITQIED
DYDDIDIEDLIPNENMVVTITHRGYIKRVPSKQYEKQKRGGKGKLAVTTYDDDFIESFFTANTHDTLMFV
TDRGQLYWLKVYKIPEGSRTAKGKAVVNLINLQAEEKIMAIIPTTDFDESKSLCFFTKNGIVKRTNLSEY
QNIRSVGVRAINLDENDELVTAIIVQRDEDEIFATGGEENLENQEIENLDDENLENEESVSTQGKMLFAV
TKKGMCIKFPLAKVREIGRVSRGVTAIKFKEKNDELVGAVVIENDEQEILSISAKGIGKRTNAGEYRLQS
RGGKGVICMKLTEKTKDLISVVIVDETMDLMALTSSGKMIRVDMQSIRKAGRNTSGVIVVNVENDEVVSI
AKCPKEENDEDELSDENFGLDL
>50S_L22 Campylobacter 50S_L22:A103V
MSKALIKFIRLSSTKARLIAREVQGMNAELAMASLKFMPNKGAKYIANAISSAVANGGFEANEVIVKSCRVDAAAVLKRF
RPRARGSASRIRKPTSHILVEVVKAEVKAEEKKTVAKKTTTTKAPAKKTTSTKKATAKKES

>nimIJ_hmm  WP_027455679.1 NimIJ family nitroimidazole resistance protein [Prevotella brevis]
MSEFREMRRKRQQLTDADSIAVLQKATSGTLALLGDNDYPYAVPISYVYDNGKLYFHSAMAGHKVDAIRR
CNKASFCVIEKDDVRPEKYTTYFRSVIAFGRIEIVEDEAEKRTIMHMMGNRFNPNHDDALQKELESGLAH
MLAIRMDIEHLTGKEAIELVRQRGGN
