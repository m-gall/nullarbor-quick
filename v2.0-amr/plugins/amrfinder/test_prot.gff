##gff-version 3
##sequence-region contig1 1-50000
contig1	.	gene	101	961	.	+	.	ID=gene1;Name=blaTEM-156
contigX	.	gene	1	501	.	+	.	ID=gene2;Name=nimIJ_hmm
contig2	.	gene	1	1191	.	+	.	ID=gene3;Name=blaPDC-114_blast
contig3	.	gene	101	802	.	+	.	ID=gene4;Name=blaOXA-436_partial
contig4	.	gene	101	1147	.	+	.	ID=gene5;Name=vanG
contig6	.	gene	31	2616	.	+	.	ID=gene6;Name=gyrA
contig7	.	gene	100	525		.	+	.	ID=gene7;Name=50S_L22
