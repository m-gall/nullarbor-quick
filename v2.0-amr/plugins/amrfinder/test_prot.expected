Target identifier	Gene symbol	Protein name	Method	Target length	Reference protein length	% Coverage of reference protein	% Identity to reference protein	Alignment length	Accession of closest protein	Name of closest protein	HMM id	HMM description
blaOXA-436_partial	blaOXA	OXA-48 family class D beta-lactamase	PARTIAL	233	265	87.92	100.00	233	WP_058842180.1	OXA-48 family carbapenem-hydrolyzing class D beta-lactamase OXA-436	NF000387.2	OXA-48 family class D beta-lactamase
blaPDC-114_blast	blaPDC	PDC family class C beta-lactamase	BLAST	397	397	100.00	99.75	397	WP_061189306.1	class C beta-lactamase PDC-114	NF000422.2	PDC family class C beta-lactamase
blaTEM-156	blaTEM-156	class A beta-lactamase TEM-156	ALLELE	286	286	100.00	100.00	286	WP_061158039.1	class A beta-lactamase TEM-156	NF000531.2	TEM family class A beta-lactamase
nimIJ_hmm	nimIJ	NimIJ family nitroimidazole resistance protein	HMM	166	NA	NA	NA	NA	NA	NA	NF000262.1	NimIJ family nitroimidazole resistance protein
vanG	vanG	D-alanine--D-serine ligase VanG	EXACT	349	349	100.00	100.00	349	WP_063856695.1	D-alanine--D-serine ligase VanG	NF000091.3	D-alanine--D-serine ligase VanG
